### ISSUE TITLE
Please make missing imports error messages give a suggestion on how to install them

### ISSUE THREAD

**Issue by jappeace** [xonsh/xonsh#2215]
saves new users the pain of figuring out how to install something, and yes you can do it in fact with pip. Also in some cases its a little obscure (z for example)

Current error message:

```
/usr/lib64/python3.4/site-packages/xonsh/__amalgam__.py:9927: ImportWarning: could not find xontrib module 'z'
  ImportWarning)
```

Desired error message:

```
/usr/lib64/python3.4/site-packages/xonsh/__amalgam__.py:9927: ImportWarning: could not find xontrib module 'z'
  ImportWarning)
Please don't fortget to install with: 'pip install xontrib-z'
```

Alternatively you could have the setup wizard install the stuff automatically, but this is probably much simpler to implement and also catches the cases of when users manually start fiddling with their config files.

**Comment by gforsyth** [comment]
Hey @jappeace -- thanks for reporting! Can you please post the output of `xonfig` please? 

And how are you enabling the `z` xontrib? On my machine, trying to load an uninstalled xontrib yields this message:

```
gil@bad_cat ~ 🐚  xontrib load powerline
The following xontribs are enabled but not installed: 
   powerline
To install them run 
    pip install xontrib-powerline
```

**Comment by jappeace** [comment]
Oh I guess I have a bug then, actually I already installed z, but I still configured promt_vi_mode to be active too, same story:

```
#( 02/19/17@12:37 )( jappie@valar-morghulis ):~
   xonsh
/usr/lib64/python3.4/site-packages/xonsh/__amalgam__.py:9927: ImportWarning: could not find xontrib module 'prompt_vi_mode'
  ImportWarning)
 ~  xonfig
+------------------+--------------------+
| xonsh            | 0.4.7              |
| Git SHA          | None               |
| Python           | 3.4.5              |
| PLY              | 3.9                |
| have readline    | True               |
| prompt toolkit   | None               |
| shell type       | readline           |
| pygments         | 2.1.1              |
| on posix         | True               |
| on linux         | True               |
| distro           | Gentoo Base System |
| on darwin        | False              |
| on windows       | False              |
| on cygwin        | False              |
| is superuser     | False              |
| default encoding | utf-8              |
| xonsh encoding   | utf-8              |
| encoding errors  | surrogateescape    |
+------------------+--------------------+
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
TypeError: cmd_handler() takes 0 positional arguments but 4 were given
Exception raised in event handler; ignored.

```

**Comment by gforsyth** [comment]
Ahh -- can you update `xonsh` and try again? We're on version `0.5.5` right now

**Comment by jappeace** [comment]
Oh, well its not in portage yet:
```

   eix xonsh
[I] dev-python/xonsh
     Available versions:  (~)0.2.6 (~)0.4.4 (~)0.4.7 {test PYTHON_TARGETS="python3_4 python3_5"}
     Installed versions:  0.4.7(23:55:55 18-02-17)(-test PYTHON_TARGETS="python3_4 -python3_5")
     Homepage:            http://xonsh.readthedocs.org/ https://github.com/scopatz/xonsh http://pypi.python.org/pypi/xonsh
     Description:         An exotic, usable shell
```



 I'll just fork the ebuild and bump the version.

**Comment by jappeace** [comment]
Okay, I don't know how to do that anymore (and its too late now to figure it out)..

Never mind the older version works except for having a not so nice error message, I'll just wait for the maintainers to bump the version.
